Xip-patch IP address fix (#2126)
kyma-project/kyma: components/xip-patch/README.md

## Usage

> The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved external IP address. If this 

Reviewer: ```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved for the external IP address. If this 
```

Reviewer 2: I would rather suggest:
```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must hold manually reserved external IP address. If this 
```

> The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is manually reserved external IP address. If this 

Reviewer: ```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which is a manually reserved, external IP address. If this 
```

> The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must hold a manually reserved, external IP address. If this 

Reviewer: ```suggestion
The patch accepts the **EXTERNAL_PUBLIC_IP** environment variable which must contain a manually reserved, external IP address. If this 
```

> variable is not set, the script tries to get external IP address from the `istio-ingressgateway` service.

Reviewer: ```suggestion
variable is not set, the script tries to get the external IP address from the `istio-ingressgateway` service.
```
